repose-aggregator/tests/functional-tests/src/test/groovy/features/core/proxy/RequestQueryParamTest.groovy

```diff
@@ -87,8 +87,11 @@ class RequestQueryParamTest extends ReposeValveTest {
 
         where:
         uriSuffixGiven                             | uriSuffixExpected                          | method
-        "/path/to/resource?key=value@%"            | "/path/to/resource?key=value%40%25"        | "GET"
-        "/path/to/resource?key=va%lu@e"            | "/path/to/resource?key=va%25lu%40e"        | "GET"
+        // todo: The '%' character breaks Jetty -- getQueryString returns the raw character while all of the
+        // getParameter methods return garbage. What do we want to do about this?
```

Comment:
they are illegal input from a url/query parameter perspective right? What happens in the system right now with them?